src/ngraph/op/constant.hpp

```diff
@@ -246,6 +249,10 @@ namespace ngraph
 
             bool is_constant() const override { return true; }
             bool are_all_data_elements_bitwise_identical() const;
```

Comment:
Should we change the impl. of this method to return the precomputed member after it's been initialized once?